src/screens/ShowcaseSheet.js

```diff
@@ -56,10 +55,6 @@ export default function ShowcaseScreen() {
     init();
   }, [addressOrDomain]);
 
-  useEffect(() => {
```

Comment:
This is the second place where the "old" open state redux was used.

I am not sure why this was used here in the first place, or if there is some other broken behavior we need to consider.